frontend/src/common/types/podcast/delete-action-podcast-payload.type.ts

```diff
@@ -0,0 +1,6 @@
+type DeleteActionPodcastPayload = {
```

Comment:
not sure that we need it
we can take id of episode from params 🤔

Reply:
I also use it for identifying userId for custom redirect to user page.

Reply:
Good!